src/Testing/TypeInferenceTestCase.php

```diff
@@ -152,24 +156,32 @@ public static function gatherAssertTypes(string $file): array
 				return;
 			}
 
+			$projectRoot = realpath(__DIR__ . '/../../');
```

Comment:
This is useless, we're still inside the PHAR. Current working directory would make more sense.